x-pack/test/security_solution_api_integration/test_suites/investigation/saved_objects/tests/draft_timeline.ts

```diff
@@ -36,7 +36,7 @@ export default function ({ getService }: FtrProviderContext) {
 
       it('returns a draft timeline template if none exists', async () => {
         const response = await supertest
-          .post('/api/timeline/_draft')
+          .post(TIMELINE_DRAFT_URL)
           .set('kbn-xsrf', 'true')
           .set('elastic-api-version', '2023-10-31')
           .send({
```

Comment:
Not in the scope of this PR but it might make sense to add the request body types here at some point :)